test/util/util.go

```diff
@@ -138,6 +139,14 @@ func SetAdmission(ctx context.Context, k8sClient client.Client, wl *kueue.Worklo
 		var newWL kueue.Workload
 		gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(wl), &newWL)).To(gomega.Succeed())
 		newWL.Status.Admission = admission
+		newWL.Status.Conditions = wl.Status.Conditions
+		apimeta.SetStatusCondition(&newWL.Status.Conditions, metav1.Condition{
+			Type:               kueue.WorkloadAdmitted,
+			Status:             metav1.ConditionTrue,
```

Comment:
it's only true if `admission != nil`, right?

Reply:
right